CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+# dbt_workday v0.2.0
+## 🚨 Breaking Changes 🚨
+- Created a surrogate key `employee_id` in `workday__employee_overview` that combines `worker_id`, `position_id`, and `position_start_date`. This accounts for edge cases like when:
```

Comment:
I think it is relevant to include this in the release notes, but what in particular makes this a breaking change?

The release should definitely be a minor `0.X.0` release, but not sure if this is a breaking change. What are your thoughts?

Reply:
Yeah, I went back and forth on this, as it isn't technically a standard breaking change. But we are updating the primary key. 

Because I was adding a new primary key, changing the grain of the model from `worker_id` to `employee_id`, it felt like a significant enough change to warrant as breaking? Or is there something else we should call it out as like 

🔑 New Primary Key 🔑 ?